src/handlers/encode-decode.ts

```diff
@@ -0,0 +1,107 @@
+import { Permit, TokenType } from "../types";
+import { RewardPermit } from "./permit-type";
+
+/**
+ * Returns a base64 encoded string containing all the permit data
+ */
+export function encodePermits(permits: Permit[]) {
```

Comment:
why do we need encode and decode? My understanding is that we will use the new permit format everywhere https://github.com/ubiquibot/permit-generation/issues/8#issuecomment-2041920173

Reply:
Correct. The reason I do this in the first place is to avoid breaking all the permits used before the update. To avoid downtime and issues I think that we should maybe make pay.ubq.fi accept old + new permits before rolling the bot update

Reply:
Related issue here: https://github.com/ubiquity/pay.ubq.fi/issues/222